packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -326,10 +326,17 @@ function resolveStyle(
   const propNamesToReprocess = Object.keys(stylesToReprocess);
   if (propNamesToReprocess.length > 0) {
     const processedStyles = processStyle(stylesToReprocess, true);
-    Object.assign(result, resolveStyle(processedStyles, options));
+    // We can end up re-processing lineHeight without a fontSize (since it might already be resolved),
+    // which generates incorrect lineHeight values. Passing the fontSize back in avoids this.
+    const fontSizeStyle =
+      result.fontSize != null ? { fontSize: result.fontSize } : null;
+    Object.assign(
+      result,
+      resolveStyle([fontSizeStyle, processedStyles], options)
+    );
```

Comment:
This is a hack to workaround a bug in how `lineHeight` is resolved. Figuring out the correct computed value for `lineHeight` involves knowing the `fontSize`, but this function can end up in a state where the `stylesToReprocess` partial contains `lineHeight` without the accompanying `fontSize` (which is in the already-computed `result` object). The quick fix (with minor perf consequence) is to forward any computed `fontSize` along with the remaining style partial when it is passed through `resolveStyle` again.